src/Memoize.jl

```diff
@@ -76,4 +76,12 @@ macro memoize(args...)
     end)
 
 end
+
+macro clear_cache(f)
+    fcachename = Symbol("##", f, "_memoized_cache")
+    esc(quote
+        Base.empty!(eval($fcachename))
```

Comment:
```suggestion
        Base.empty!($fcachename)
```

Reply:
Apologies for misreading the suggestion!